assembly/fabric8-stacks/src/main/resources/stacks.json

```diff
@@ -64,6 +64,152 @@
       "mediaType": "image/svg+xml"
     }
   },
+  {
+    "id": "che7-preview",
+    "creator": "ide",
+    "name": "Che 7 Preview",
+    "description": "Workspace.next sidecars and Theia as IDE",
+    "scope": "general",
+    "tags": [
+      "ws.next",
+      "Theia",
+      "Java",
+      "JDK",
+      "Node.JS",
+      "NPM",
+      "Yeoman"
+    ],
+    "components": [
+      {
+        "name": "Centos",
+        "version": "7"
+      },
+      {
+        "name": "OpenJDK",
+        "version": "1.8.0_181"
+      },
+      {
+        "name": "NodeJS",
+        "version": "8.12.0"
+      },
+      {
+        "name": "NPM",
+        "version": "6.4.1"
+      }
+    ],
+    "workspaceConfig": {
+      "name": "che7",
+      "defaultEnv": "default",
+      "environments": {
+        "default": {
+          "machines": {
+            "ws/dev": {
+              "attributes": {
+                "memoryLimitBytes": "536870912"
```

Comment:
We don't need both an attribute and memory limit in the recipe. Can we remove one of them?

Reply:
The same for a second stack